.babelrc

```
@@ -1,3 +1,5 @@
 {
-  "presets": ["env"]
+  "presets": [
```

This can be shortened to
```suggestion
  "presets": ["@babel/env"]
```